### ISSUE TITLE
handle 404 errors in react routing

### ISSUE THREAD

**Issue by giocalitri** [mitodl/micromasters#602]
depends on #440 

@noisecapella was saying that react should force a reload when a route is not found to have the server  return the 404 via template.

**Comment by aliceriot** [comment]
is this a duplicate of #481?

**Comment by noisecapella** [comment]
Yes, looks like it

**Comment by noisecapella** [comment]
Closing as duplicate of #481
